packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -58,14 +58,15 @@ export type GeneratorConfig = FormConfig<any> | GridConfig<any> | TabsConfig;
 
 export type GeneratorReturn = { code: string; gqlDocuments: Record<string, string> };
 
-export async function runFutureGenerate() {
+export async function runFutureGenerate(specificFile?: string) {
```

Comment:
How about changing that to support multiple files? Or even pass the glob pattern directly?

Reply:
For now I only needed to rerun single files for debugging purposes, or to update a single file. generating a single-file could even be handy for scaffolding purposes.

what would be the use case for multiple files or a custom glob pattern?

Reply:
> what would be the use case for multiple files or a custom glob pattern?

I was thinking about being able to faster generate only parts while developing a feature. For instance, if I'm currently developing the products module, it could be handy to regenerate all products-related parts with `admin-generator src/products/**/*.cometGen.ts`

Reply:
I changed it to support glob-pattern.